src/app/core/components/login-page/login-page.component.ts

```diff
@@ -21,16 +24,41 @@ export class LoginPageComponent implements OnInit {
   ngOnInit() {
   }
 
+  ngOnDestroy() {
+    if (this.aSub) {
```

Comment:
Naming conversion for subscribtions is adding $ at the end of the name. You should write something like authentication$

Reply:
changed naming